libgnomekbd: broken

```
=> libgnomekbd-3.6.0_4: running do_build ...
make  all-recursive
make[1]: Entering directory '/builddir/libgnomekbd-3.6.0'
Making all in libgnomekbd
make[2]: Entering directory '/builddir/libgnomekbd-3.6.0/libgnomekbd'
  GEN      gkbd-indicator-marshal.c
/bin/sh: glib-genmarshal: command not found
Makefile:1209: recipe for target 'gkbd-indicator-marshal.c' failed
make[2]: *** [gkbd-indicator-marshal.c] Error 127
make[2]: *** Waiting for unfinished jobs....
  GEN      gkbd-keyboard-drawing-marshal.h
/bin/sh: glib-genmarshal: command not found
Makefile:1212: recipe for target 'gkbd-keyboard-drawing-marshal.h' failed
make[2]: *** [gkbd-keyboard-drawing-marshal.h] Error 1
make[2]: Leaving directory '/builddir/libgnomekbd-3.6.0/libgnomekbd'
Makefile:473: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/builddir/libgnomekbd-3.6.0'
Makefile:379: recipe for target 'all' failed
make: *** [all] Error 2
=> ERROR: libgnomekbd-3.6.0_4: failed to run do_build() at line 13.
```

--- a/srcpkgs/libgnomekbd/template
+++ b/srcpkgs/libgnomekbd/template
@@ -4,7 +4,7 @@ version=3.6.0
 revision=4
 build_style=gnu-configure
 configure_args="--disable-static"
-hostmakedepends="pkg-config intltool gobject-introspection"
+hostmakedepends="pkg-config intltool gobject-introspection glib-devel"
 makedepends="libxml2-devel gtk+3-devel>=3.6.0_2 libxklavier-devel>=5.2.1_1"
 depends="desktop-file-utils"
 short_desc="Gnome keyboard configuration library"

